packages/tailwind-preset/src/tailwind.css

```
@@ -194,7 +194,7 @@
  * ======================================== */
 @variant dark (&:where(.dark, .dark *));
 
-@custom-variant sentinel-intersect ([data-sentinel][data-intersect] + &);
+@custom-variant sentinel-intersect ([data-sentinel][data-intersect] + &, &:has(+ [data-sentinel][data-intersect]));
```

Stylelint (17.13.0)

[error] Unexpected unknown at-rule "`@custom-variant`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)